APIManager.js

```diff
@@ -0,0 +1,119 @@
+class APIManager {
+    constructor() {
+        this.data = {
+            user: {
+                firstName: '',
+                lastName: '',
+                pictue: '',
+                city: '',
+                state: ''
+           },
+           quote: '',
+           pokemon: {
+               image: '',
+               name: ''
+           },
+           meat: '',
+           friends: [
+               {firstName: '', lastName: ''},
+               {firstName: '', lastName: ''},
+               {firstName: '', lastName: ''},
+               {firstName: '', lastName: ''},
+               {firstName: '', lastName: ''},
+               {firstName: '', lastName: ''}
+            ]
+
+        },
+        
+        this.finishIn4 = 0;
+    };
+
+    getUser = func => {
+        $.ajax({
+            method: 'GET',
+            url: 'https://randomuser.me/api?results=7',
+            dataType: 'json',
+            success: data => {
+                this.data.user.firstName = data.results[0].name.first;
+                this.data.user.lastName = data.results[0].name.last;
+                this.data.user.picture = data.results[0].picture.medium;
+                this.data.user.city = data.results[0].location.city;
+                this.data.user.state = data.results[0].location.state;
+
+                for (let i = 1; i < data.results.length; i++) {
```

Comment:
use the `.map()` array method. there are not many times you will need to use loops again